lib/pages/main_menu.dart

```diff
@@ -170,6 +170,224 @@ class _MainMenuState extends State<MainMenu> {
                                     child: Text('Sources',
                                         style: new TextStyle(color: purple.shade500, fontSize: 20))
                                 ),
+                                ListView(
+                                  shrinkWrap: true,
```

Comment:
add a scrollPhysics attribute so that this part of the overlay is also scrollable